client/blocks/importer/wordpress/import-everything/pre-migration/index.tsx

```diff
@@ -100,6 +104,9 @@ export const PreMigrationScreen: React.FunctionComponent< PreMigrationProps > =
 	const { isLoading: isAddingTrial } = useAddHostingTrialMutation( {
 		onSuccess: () => {
 			setQueryTargetSitePlanStatus( 'fetching' );
+			recordFreeHostingTrialStarted(
+				migrationTrackingProps.is_migrate_from_wp ? FlowNames.SiteMigration : FlowNames.NewSite
```

Comment:
This call may not be necessary, as this happens after the import, while the first call happens when the trial is acknowledged. I can't get this one to trigger, but I think it's better to have it earlier in the flow.

Reply:
I took this away for now, and rather added it to the flows in the submit method under the `trialAcknowledge` step. As this is used for intent/targeting ads, so we have a shorter feedback loop from click -> trial start.